src/libproc_macro/qquote.rs

```diff
@@ -0,0 +1,262 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! This file contains the implementation internals of the quasiquoter provided by `quote!`.
```

Comment:
Could you give a few more details please? E.g., that it outputs tokens, the syntax for unquoting, what can be unquoted, etc. Maybe a simple example.